data/transport missions.txt

```diff
@@ -1951,3 +1951,384 @@ ship "Science Drone"
+
+
+
+mission "Lost Boy 3"
+	landing
+	name `Bring Tod to <planet>`
+	description `Now that Tod's mother knows he is safe, bring him to <destination> so that he may start his mining job to support him and his mother.`
+	destination "Placer"
+	to offer
+		has "Lost Boy 2: done"
+	
+	on offer
+		payment 952
+		conversation
+			`Tod's mother almost tears up at the sight of him as you approach. "Oh, my baby boy is alive!" she exclaims, running up and hugging Tod as his face reddens with embarrassment. "Are you hurt? Are you okay? Do you think you'll be fine?" Tod's mother asks, kissing his cheeks between each question.`
+			`	"I'm fine, Mom," Tod says in an annoyed tone of voice. Some things never change.`
+			`	The mother gives you a handful of credit chips. Upon counting them, you find that they are only worth a total of <payment>. "I know it isn't much for the work you've done, Captain, but it's all I have to give you in return."`
+			choice
+				`	"Don't worry. "`
+				`	"You're lucky I agreed to help you in the first place."`
+					goto money
+			
+			`	"This is just so wonderful. There aren't many people around here who would do such a thing for such little pay."`
+				goto end
+			
+			label money
+			`	"I know, I know, and I'm eternally grateful for that, Captain. There aren't many people around here who would do such a thing for such little pay."`
+			
+			label end
+			`	"Hold on," Tod chimes in. "I still need to get to <planet>."`
+			`	"Of course!" his mother yells out. "Captain, would you please make sure that Tod has a safe passage to <destination>?"`
+			`	"And make sure I don't become a slave this time," Tod quips jokingly.`
+			choice
+				`	"Okay, I'll take Tod to <planet>."`
+					accept
+				`	"Sorry, I'm not headed in that direction."`
+					decline
+	
+	npc
+		government "Pirate"
+		personality staying plunders disables
+		system "Al Dhanab"
+		ship "Falcon (Heavy)" "Cygnet's Slaver"
+	
+	on complete
+		dialog `You wish Tod Copper the best of luck on <planet>. "Thank you, Captain. Hopefully I can repay you one day. As cliche as it sounds, I owe you my life."`
+
+
+
+mission "Paradise Fortune 1"
+	minor
+	landing
+	name `Flee to <planet>`
+	description `Flee from <origin> to <destination> with the girl you let on to your ship.`
+	source
+		planet "Follower" "Mainsail"
+	destination
+		distance 1 1
+	passengers 1
+	to offer
+		has "main plot completed"
+		random < 50
+	
+	on offer
+		conversation
+			`You land your ship on an empty landing pad situated within walking distance of the <origin> shipyard. As with all Paradise Worlds, the air is just the right temperature when you step out of your ship, and a light breeze rustles the leaves of the trees planted around the spaceport.`
```

Comment:
If this is rephrased to remove the landing / leaving ship reference, then this mission can be a spaceport mission and not a `landing` one.